kolibri/plugins/base.py

```diff
@@ -131,6 +142,31 @@ def api_url_module(self):
         """
         return self._return_module("api_urls")
 
+    def settings_module(self):
+        """
+        Return a settings module, containing Django settings that this
+        module wants to apply.
+
+        For settings that take a tuple or list, these will be appended to the value from
+        the base settings module set through conventional Django means.
+
+        By default settings.py will be automatically discovered, if you wish to
+        override this behaviour, override this method.
+        """
+        return self._return_module("settings")
```

Comment:
This is where I get most concerned.

Reply:
This is a convenience method - this can be directly defined as a particular module if so desired.

Could also add a try catch around the import in the `_return_module` method to avoid issues, but I thought it was better to raise errors.